packages/devextreme/testing/systemjs-builder.js

```diff
@@ -243,15 +242,6 @@ const transpileJsVendors = async() => {
         }),
     );
 
-    await Promise.all(
```

Comment:
Do you have any ideas, why we even need this file in DevExtreme? I've found it in additional qunit build test task, but the purpose for me is unclear for now

Reply:
Default system.js builder doesn't support SCP. For more details see https://github.com/DevExpress/DevExtreme/pull/24219